README.md

```diff
@@ -1,3 +1,4 @@
-All content from this repository has been moved to the [Gradle User Manual](https://docs.gradle.org/current/).
+# Gradle Guides Plugin
 
-The Gradle User Manual sources are located here: https://github.com/gradle/gradle/tree/master/subprojects/docs
+All Gradle documentation content from this repository has been moved to the
+[Gradle User Manual](https://docs.gradle.org/current/).
```

Comment:
💅 Would be nice to expand readme a bit:
* What is the purpose of this plugin
* Is it gradle-only
* How to update the version/deploy
* etc

Reply:
The readme from the plugin project covers most of this IMO, and I move it to the top as requested below

Reply:
LGTM!